### ISSUE TITLE
JUnit4to5Migration only works with POM; not via command line

### ISSUE THREAD

**Issue by crehn** [openrewrite/rewrite-testing-frameworks#288]
```
git clone https://github.com/openrewrite/spring-petclinic-migration.git
cd spring-petclinic-migration
mvn org.openrewrite.maven:rewrite-maven-plugin:4.37.0:run \
  -Drewrite.recipeArtifactCoordinates=org.openrewrite.recipe:rewrite-testing-frameworks:1.31.0 \
  -DactiveRecipes=org.openrewrite.java.testing.junit5.JUnit4to5Migration
```
produces
```
[WARNING] java.lang.IllegalArgumentException: Unable to find runtime dependencies beginning with: 'junit-jupiter-api'
[WARNING] java.lang.IllegalArgumentException: Unable to find runtime dependencies beginning with: 'apiguardian-api', 'junit-jupiter-api'
[WARNING] java.lang.IllegalArgumentException: Unable to find runtime dependencies beginning with: 'junit-jupiter-api'
[WARNING] java.lang.IllegalArgumentException: Unable to find runtime dependencies beginning with: 'apiguardian-api', 'junit-jupiter-api'
[ERROR] The recipe produced an error. Please report this to the recipe author.
```

Doing virtually the same by adding the plugin to pom.xml and running `mvn rewrite:run` works fine, though.

**Comment by timtebeek** [comment]
Sounds similar to https://github.com/openrewrite/rewrite-maven-plugin/issues/437. Would you agree?

**Comment by crehn** [comment]
Yeah, it sounds similar. You are right.

This bug took me like half a day to come up with the workaround. The docs say that command line is fine and as this is far more convenient, I hadn't tried the pom approach in the first place. If fixing this bug takes some time, maybe you could at least place a hint in the docs. I almost gave up on this great tool and it would be sad if such problems scare away people.

**Comment by tkvangorder** [comment]
This issue was fixed as part of the following PR in the rewrite-maven-plugin.

https://github.com/openrewrite/rewrite-maven-plugin/pull/445

This fix will be in 4.38.0 of the rewrite-maven-plugin and we anticipate doing a new release tomorrow.
